### ISSUE TITLE
symbl.cc Accessability issue

### ISSUE THREAD

**Issue by BANOnotIT** [symbl-cc/symbl-data#169]
Cards with some selectors doesn't comply with any WCAG.
- `.hover-color:nth-child(8n+5) a:hover` [doesn't comply](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=00FFF7)
- `.hover-color:nth-child(8n+6) a:hover` [doesn't comply](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FFFF00)
- `.hover-color:nth-child(8n+7) a:hover` [doesn't comply](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FE7809)

**Comment by asanov** [comment]
Yes, thank you, we know about this issue and will fix it till the end of the month.
